### PR TITLE
docs: Add instructions on using Minikube as an alternative to K3D

### PR DESCRIPTION
**Pull request by terrytangyuan**
Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [x] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed the CLA.
* [x] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My builds are green. Try syncing with master if they are not. 
* [x] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: docs/running-locally.md [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+We recommend using [K3D](https://k3d.io/) to set up the local Kubernetes cluster since this will allow you to test RBAC set-up and is fast. You can set-up K3D to be part of your default kube config as follows:
 
     cp ~/.kube/config ~/.kube/config.bak
     cat $(k3d get-kubeconfig --name='k3s-default') >> ~/.kube/config
+    
+Alternatively, you can use [Minikube](https://github.com/kubernetes/minikube) to set up the local Kubernetes cluster. Once a local Kubernetes cluster has started via `minikube start`, your kube config will use Minikube's context automatically.
+
+Useful:
+
+* For a PS1 prompt showing your current kube context: kube-ps1 to help.  `brew install kube-ps1`
```
Comment (simster7) [inline docs/running-locally.md:22]: Could you explain why this is needed?

Comment (terrytangyuan) [reply docs/running-locally.md:22]: This was in the doc before. I am not sure why it's necessary either. Should I just delete it?

Comment (alexec) [reply docs/running-locally.md:22]: simpler is better - > please delete

Comment (terrytangyuan) [reply docs/running-locally.md]: Thanks. Done.

**Comment by alexec** [comment]
LGTM

**Pull request merged**